raml-model/src/main/java/io/vrap/rmf/raml/persistence/RamlResource.java

```diff
@@ -54,15 +54,14 @@ protected void doLoad(final InputStream inputStream, final Map<?, ?> options) th
 
             try {
                 rootConstructor.construct(parser, resourceScope);
-                validate();
             } catch (final Exception e) {
```

Comment:
Before we already performed the validation on a partially loaded model and this then caused problems.